File: common/NetPort.py
def getEnvPort():
    try:
        f = open('/tmp/obotcha_test_suit_port.txt', 'r')
        list1 = f.readlines()
        if f:
            f.close()
        
        if len(list1) <= 0:
            return 4000

        return int(list1[0])
    except IOError:
        return 4000 
    
def setEnvPort(port):
    if port > 60000:
        port = 4000
    fo = open("/tmp/obotcha_test_suit_port.txt", "w")
    fo.write(str(port))
    fo.close()

def resetEnvPort():
    setEnvPort(4000)

File: common/NetPort.go
package testnet

import (
    "fmt"
    "io/ioutil"
    "strconv"
    "os"
    "log"
)

func GetEnvPort() int{
    data, err := ioutil.ReadFile("/tmp/obotcha_test_suit_port.txt")
    if err != nil {
        fmt.Println("File reading error", err)
        return 2001
    }
    int5, err := strconv.Atoi(string(data))
    return int5
}

func SetEnvPort(port int) {
    if err := ioutil.WriteFile("/tmp/obotcha_test_suit_port.txt", []byte(strconv.Itoa(port)), 0666); err != nil{
        os.Exit(111)
        log.Println(err.Error())
    }
}

func ResetEnvPort() {
    SetEnvPort(2001);
}